### ISSUE TITLE
jsk permtrace bug

### ISSUE THREAD

**Issue by Atharv-Agarwal** [Gorialis/jishaku#135]
## Summary

The `jsk permtrace` doesnt work as expected

## Reproduction steps

Use the permtrace, It only returns the permissions for the guild owner/role or users with admin perms. 
`??jsk permtrace <channel_id> <user_id>`
I also tried using a `<role_id>` but that gave me the same error.

## Expected results

To show the results of the command, ie the users permissions in the channel. 
## Actual results

```
Traceback (most recent call last):
  File "/app/.heroku/python/lib/python3.9/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/app/.heroku/python/lib/python3.9/site-packages/jishaku/features/guild.py", line 138, in jsk_permtrace
    if overwrite.is_role() and overwrite.id in role_lookup:
AttributeError: '_Overwrites' object has no attribute 'is_role'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/app/.heroku/python/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 939, in invoke
    await ctx.command.invoke(ctx)
  File "/app/.heroku/python/lib/python3.9/site-packages/discord/ext/commands/core.py", line 1348, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "/app/.heroku/python/lib/python3.9/site-packages/discord/ext/commands/core.py", line 863, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "/app/.heroku/python/lib/python3.9/site-packages/discord/ext/commands/core.py", line 94, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: '_Overwrites' object has no attribute 'is_role'
```

## Checklist

- [x] I have updated discord.py and jishaku to the latest available versions and have confirmed that this issue is still present
- [x] I have searched the open issues for duplicates
- [x] I have shown the entire traceback, if possible
- [x] I have removed my token from display, if visible

## System information

Jishaku v2.3.1, discord.py 1.7.3, Python 3.9.6

**Comment by Gorialis** [comment]
Whoops, looks like I slipped in a 2.0a-only change without realizing it. I'll have to shim for it but should be fixed soon enough.

**Comment by Gorialis** [comment]
This should be fixed in 2.3.2, which will be available from pip likely within the next hour.
